robotbase/src/main/java/frc/robot/subsystems/AlgaePivot.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+
+import com.revrobotics.spark.SparkMax;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ALGAE_PIVOT;
+import frc.robot.Constants.CAN_ID;
+
+public class AlgaePivot extends SubsystemBase {
+    private SparkMax m_leftMotor;
+    private SparkMax m_rightMotor;
```

Comment:
We should only have 1 motor for this subsystem

Reply:
False, it is two motors as of when I asked last night.